src/main/java/uk/gov/digital/ho/hocs/search/application/aws/config/ElasticSearchConfiguration.java

```diff
@@ -9,18 +9,13 @@
 import org.springframework.context.annotation.Profile;
 
 @Configuration
-@Profile({"localelastic"})
+@Profile({"!local"})
 public class ElasticSearchConfiguration {
 
-    @Value("${elasticsearch.host}")
-    private String host;
-
-    @Value("${elasticsearch.port}")
-    private int port;
-
     @Bean(destroyMethod = "close")
-    public RestHighLevelClient client() {
-
+    public RestHighLevelClient client(
+            @Value("${aws.es.host}") String host,
```

Comment:
If you wanted to be really cool, we could just have an [configuration class](https://docs.spring.io/spring-boot/docs/2.0.x/reference/html/using-boot-configuration-classes.html) for this